pokepro/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import pokeball from "../../assets/assets/pokeball-icon.png";
+import "./header.css";
+const Header: React.FC = () => {
+  return (
```

Comment:
If the component is just returning the jsx we can write it like this.

```suggestion
  const Header: React.FC = () => (<header></header>)
```

Without the return